### PR TITLE
Cache using multicache module

### PR DESCRIPTION
**Pull request by bargulg**
Multicache module source code:
https://github.com/bargulg/multicache

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,8 @@
     'ipdb',
     'coveralls',
     'sphinx',
-    'vcrpy'
+    'vcrpy',
+    'multicache'
```
Comment (adback03) [inline setup.py]: multicache won't be needed here, since it is specified in the install_requires section

**Comment by bargulg** [comment]
closing until we get a reasonable way to configure the cache

File: tests/test_card.py [hunk 2]
```diff
@@ -48,6 +48,18 @@ def test_all_with_params_return_cards(self):
...
+        # this is used to test cache
+        with vcr.use_cassette('fixtures/legendary_elf_warriors.yaml'):
+            cards = Card.where(supertypes='legendary') \
+                .where(subtypes='elf,warrior') \
+                .all()
+            self.assertTrue(len(cards) >= 13)
+            cards = Card.where(supertypes='legendary') \
+                .where(subtypes='elf,warrior') \
+                .all()
+            self.assertTrue(len(cards) >= 13)
```
Comment (silasary) [inline tests/test_card.py]: This doesn't actually test the cache.

Comment (bargulg) [inline tests/test_card.py]: I know, this is just a basic sanity check to make sure that the cache doesn't break anything.
Multicache module itself has its own tests

**Pull request closed without merging**